spec/support/tahi_helper_methods.rb

```diff
@@ -79,6 +79,11 @@ def with_aws_cassette(name)
     end
   end
```

Comment:
Can we get rid of this helper now?

Reply:
It's still used. I only created the `insert_aws_cassette` helper to work around an issue in rspec-capybara. capybara does not work with `around` blocks so in https://github.com/Aperta-project/Aperta/pull/4091/files#diff-01359a56dac986586ece9a7c9bb23676R81 where I abstracted out the various combinations of preprint flag, etc I needed to use `before` and `after` to insert/eject VCR cassettes.

Reply:
https://github.com/teamcapybara/capybara/issues/1029

Reply:
Ah, got it